storage-provider/server/src/storage.rs

```diff
@@ -80,14 +80,21 @@ pub async fn start_upload_server(
 fn configure_router(state: Arc<StorageServerState>) -> Router {
     #[cfg(feature = "delia")]
     fn config_delia(state: Arc<StorageServerState>) -> Router {
+        use axum::extract::DefaultBodyLimit;
+
         let cors = CorsLayer::new()
             .allow_origin(Any)
             .allow_methods([Method::GET, Method::POST, Method::PUT, Method::OPTIONS])
             .allow_headers([header::CONTENT_TYPE])
             .max_age(std::time::Duration::from_secs(3600));
 
         Router::new()
-            .route("/upload/:cid", put(upload))
+            .route(
+                "/upload/:cid",
+                put(upload)
+                    // Without disable we can't handle
+                    .layer(DefaultBodyLimit::disable()),
```

Comment:
Probably there is a reason for the default body limit, meaning DoS attacks. I'm happy to disable it now anyways I suppose.